.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
```

Comment:
This workflow is for building the project. Do you have another for running tests? I see that you have few tests.

Reply:
Build includes running tests for both platforms

Reply:
CI runs on ephemeral node, so it does not store resulted artifacts... Later I guess we will add separated manual pipeline to release.

Reply:
Just to confirm `gradle build` both compiles and runs tests? That's a bit confusing.

Reply:
Right... Honestly I am not an expert with gradle (working with maven mostly) so empirically I defined that `test` goal does not run test for iOS platform for some reason.